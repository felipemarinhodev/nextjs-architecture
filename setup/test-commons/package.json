{
  "name": "@felipemarinhodev/test-commons",
  "version": "0.0.0",
  "main": "index.js",
  "private": true,
  "author": "Felipe Marinho <npm@felipemarinho.dev> (https://felipemarinho.dev/)",
  "license": "MIT",
  "files": [
    "base.ts"
  ],
  "scripts": {
    "test": "echo 'Not implemented'",
    "lint": "echo 'Not implemented'"
  },
  "devDependencies": {
    "@babel/core": "^7.18.9",
    "@babel/preset-env": "^7.18.9",
    "@babel/preset-react": "^7.18.6",
    "@babel/preset-typescript": "^7.18.6",
    "babel-jest": "^28.1.3",
    "jest": "27.0.0",
    "ts-node": "^10.9.1"
  }
}
