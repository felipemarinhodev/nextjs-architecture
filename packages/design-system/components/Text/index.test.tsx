import { Text } from ".";
import { customRender } from "@felipemarinhodev/test-commons/react-testing-library";

const render = customRender();

describe("<Text />", () => {
  it("render h1 tag", () => {
    const { container } = render(<Text tag="h1">Simple text</Text>);
    expect(container).toMatchSnapshot();
  });
});
