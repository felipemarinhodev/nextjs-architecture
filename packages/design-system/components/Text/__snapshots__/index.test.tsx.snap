// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> render h1 tag 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <h1
    class="c0"
  >
    Simple text
  </h1>
</div>
`;
